spec/lib/services/sku-pack-service-spec.js

```diff
@@ -107,8 +117,22 @@ describe("SKU Pack Service", function() {
                 destroy: sinon.stub()
             };
 
+            waterline.templates = {
```

Comment:
Since you mock a dummy templates property to  the waterline service, if not remove it from waterline, then other unit test will see the same dummy templates property as well. This may impact some unit-test (if we have defined one), like check whether the waterline service has a function 'create'.